wdl2cwl/main.py

```diff
@@ -725,29 +722,19 @@ def get_expr(self, wdl_expr: WDL.Expr.Base) -> str:
     def get_expr_string(self, wdl_expr_string: WDL.Expr.String) -> str:
         """Translate WDL String Expressions."""
         if wdl_expr_string.literal is not None:
-            return f'"{wdl_expr_string.literal.value}"'
-        string = ""
+            return str(wdl_expr_string.literal)
```

Comment:
@mr-c couldn't we call `return str(get_literal_value(wdl_expr_string))` here? That way we wouldn't have to call `get_literal_value` in the code directly, but instead centralize string translations in this `get_expr_string`.

Reply:
Feel free to try it and let me know? My memory is that the result is different. This way preserves the original quotes, if I remember correctly

Reply:
Moved to follow-up issue so it doesn't block this PR: https://github.com/common-workflow-lab/wdl-cwl-translator/issues/185

:+1: